runtime/layers/fpm-dev/Dockerfile

```diff
@@ -0,0 +1,6 @@
+ARG LAYER_IMAGE
+FROM $LAYER_IMAGE
+
+COPY ./php-fpm.conf /opt/bref/etc/php-fpm.conf
+EXPOSE 9000
+CMD /opt/bin/php-fpm -F -y /opt/bref/etc/php-fpm.conf
```

Comment:
Could you use the readable version for the options? Like here:

https://github.com/brefphp/bref/blob/7127de2e15abe1f8bd3235adc4ba30b64076172f/src/Runtime/PhpFpm.php#L68

That way we can understand easily what they mean.